app/controllers/patients_controller.rb

```diff
@@ -1,9 +1,15 @@
 # Create and update patients, plus the main patient view in edit
 class PatientsController < ApplicationController
+  before_action :confirm_user_has_data_access, only: [:index]
   before_action :find_patient, only: [:edit, :update]
   rescue_from Mongoid::Errors::DocumentNotFound,
               with: -> { redirect_to root_path }
 
+  def index
+    @patients = Patient.all
+    respond_to :csv
```

Comment:
I think given the patterns we're going to have elsewhere in the app (thinking of the PDF generation PR in particular, here), it kinda might make sense to cut out the view and just do the `CSV.generate` stuff in the patient model, and then `send_data` that? That would be consistent, at least. (Sorry for the back and forth on this!)